package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@suffolklitlab/alkiln",
-  "version": "4.10.1-pre-cli",
+  "version": "4.11.0",
```

Comment:
No need to change this now, but for the future - avoid adjusting version numbers in PRs. That should be adjusted when the actual publishing happens. That's because by the time this gets published, this version number might not be correct anymore depending on what else happens in the meantime.